src/components/Contact.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
Same unused code left here, kindly remove unused lines of code to keep your codebase simple thereby implementing the **KISS** coding practice.